components/inputs/input-time.js

```diff
@@ -92,13 +123,40 @@ class InputTime extends LitElement {
 
 	render() {
 		const input = html`
-			<input
-				aria-label="${ifDefined(this._getAriaLabel())}"
-				@change="${this._handleChange}"
-				class="d2l-input"
-				?disabled="${this.disabled}"
-				@keypress="${this._handleKeypress}"
-				.value="${this._formattedValue}">
+			<d2l-dropdown>
+				<div
+					id="combo-container"
+					role="combobox"
+					aria-owns="time-menu"
+					class="d2l-dropdown-opener"
+					aria-expanded="false">
+					<input
+						aria-controls="time-menu"
+						aria-label="${ifDefined(this._getAriaLabel())}"
+						@change="${this._handleChange}"
+						class="d2l-input"
+						?disabled="${this.disabled}"
+						@keypress="${this._handleKeypress}"
+						.value="${this._formattedValue}">
+				</div>
+				<d2l-dropdown-menu id="dropdown" no-padding-footer min-width="195">
+					<d2l-menu
+						id="time-menu"
```

Comment:
We don't use `ID`s a lot because for browsers who don't support shadow DOM all of this gets slammed into the same DOM tree, and then multiple instances of a control result in duplicate IDs. Not sure if that's actually a problem or not, but one strategy we've used is to just generate ids (there's a helper for it).